.github/workflows/deno-build.yml

```diff
@@ -2,8 +2,13 @@ name: deno.land Continuous Deployment
 
 on:
   push:
+    branches:
+      - main
     tags:
       - '@slack/web-api*'
+      - '!@slack/web-api*nextGen*'
```

Comment:
Is this the best syntax? I didn't see anything specific in the docs about using asterisks within a string to detect something that contains a certain word but may have words or characters before or after it, but that's what i'm trying to do here with `nextGen`, `Beta`, and `beta`.

Reply:
Yeah I think that's fine.

It is maybe worth testing the matching syntax out on your own fork (and e.g. just commenting out the run steps or changing them to do something harmless like `echo` some text), just to make sure that pushing different tags and to different branches triggers the workflow as you expect.